app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -137,7 +141,7 @@ class HomeViewModel(
       // TODO(#936): Optimise this as part of recommended stories.
       ongoingStoryList.olderStoryList
     }
-    return storyList.take(R.integer.promoted_story_list_limit)
```

Comment:
Ah wow, nice catch. Can we make sure there's a test to cover this?

Reply:
Done

Added it in `HomeActivityTest` using the `StoryProgressTestHelper`
Alternatively worked on writing a `HomeViewModelTest` but this proved more difficult as we only had access to the view model's live data and would require manipulating the environment for creating the live data. Let me know what you think -- I think this ultimately achieves the same goal, testing that we only show x promoted stories at most in the home activity.

Reply:
So long as the test fails when the original  issue is checked in, that sounds good to me. Did you verify this?

Reply:
Oh yes it does fail when in the original, forgot to mention that!

Reply:
Great--thanks!